docs/MAINTAINERS.md

```diff
@@ -1,52 +1,180 @@
+# Build CLI
+./scripts/build/docker/run-command.sh -r x64 -s . -c "make electron-develop && make RELEASE_TYPE=production cli-installer-tar-gz"
+
+# x86
+
+# Build Debian packages
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-debian"
+# Build RPM packages
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-redhat"
+# Build AppImages
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-appimage"
+# Build CLI
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production cli-installer-tar-gz"
+```
 
-- Upgrade the `electron` dependency version in `package.json` to an *exact
-  version* (no `~`, `^`, etc).
+#### Mac OS
+
+**ATTENTION:** For production releases you'll need the code-signing key,
+and set `CSC_NAME`, `CSC_LINK`, and `CSC_KEY_PASSWORD` to generate signed binaries on Mac OS.
+
+**NOTE:** The CLI is not code-signed for either at this time.
+
+```bash
+make electron-develop
+
+# Build the CLI
+make RELEASE_TYPE=production cli-installer-tar-gz
+# Build the zip
+make RELEASE_TYPE=production electron-installer-app-zip
+# Build the dmg
+make RELEASE_TYPE=production electron-installer-dmg
+```
+
+#### Windows
+
+**ATTENTION:** For production releases you'll need the code-signing key,
+and set `CSC_NAME`, `CSC_LINK`, and `CSC_KEY_PASSWORD` to generate signed binaries on Windows.
+
+**NOTE:** The CLI is not code-signed for either at this time.
+
+```bash
+make electron-develop
+
+# Build the CLI
+make RELEASE_TYPE=production cli-installer-zip
+# Build the Portable version
+make RELEASE_TYPE=production electron-installer-portable
+# Build the Installer
+make RELEASE_TYPE=production electron-installer-nsis
+```
+
+### Uploading packages to Bintray
+
+```bash
+export BINTRAY_USER="username@account"
+export BINTRAY_API_KEY="youruserapikey"
+```
+
+```bash
+./scripts/publish/bintray.sh -c "etcher" -t "production" -v "1.2.1" -o "resin-io" -p "debian" -y "debian" -r "x64" -f "dist/etcher-electron_1.2.1_amd64.deb"
+./scripts/publish/bintray.sh -c "etcher" -t "production" -v "1.2.1" -o "resin-io" -p "debian" -y "debian" -r "x86" -f "dist/etcher-electron_1.2.1_i386.deb"
+./scripts/publish/bintray.sh -c "etcher" -t "production" -v "1.2.1" -o "resin-io" -p "redhat" -y "redhat" -r "x64" -f "dist/etcher-electron-1.2.1.x86_64.rpm"
+./scripts/publish/bintray.sh -c "etcher" -t "production" -v "1.2.1" -o "resin-io" -p "redhat" -y "redhat" -r "x86" -f "dist/etcher-electron-1.2.1.i686.rpm"
+```
+
+### Uploading binaries to Amazon S3
+
+```bash
+export S3_KEY="..."
+```
+
+```bash
+./scripts/publish/aws-s3.sh -b "resin-production-downloads" -v "1.2.1" -p "etcher" -k $(date +"%Y-%m-%d") -f "dist/<filename>"
```

Comment:
The `-k` option is only used for snapshot builds

Reply:
Aye, removed!